.gitignore

```diff
@@ -6,4 +6,3 @@ node_modules
 pnpm-debug.log
 
 /dist
-!/dist/index.html
```

Comment:
Please keep this line. It allows tracking changes to `dist/index.html` during development.

Reply:
I might be wrong, but now dist should be completely empty and dedicated to builded assets.
The dev should be made in /index.html